### PR TITLE
Fix setting dev server port

### PR DESCRIPTION
**Pull request by westonganger**
Fixes settings ordering issue described in #36

### CODE REVIEW

File: lib/frontman/commands/serve.rb [hunk 1]
```diff
@@ -75,6 +75,35 @@ def serve
...
+      num_retries = Frontman::Config.get(:port_retries, fallback: 3)
+
+      port_retry_strategy = Frontman::Config.get(
+        :port_retry_strategy,
+        fallback: ->(p) {
+          port_in_use = false
+
+          (1 + num_retries).times do
+            begin
+              port_in_use = Socket.tcp('localhost', p, connect_timeout: 3) { true }
```
Comment (westonganger) [inline lib/frontman/commands/serve.rb:89]: Getting line length errors here again. 83/100. Would love that 100 char line limit increase 😉

Comment (DevinCodes) [reply lib/frontman/commands/serve.rb:89]: Okay, feel free to set the limit to 100, it seems to be a recurring issue indeed and if we can prevent that this way let's do that 💯

**Comment by westonganger** [comment]
Apparently today there is new failures for some other rules. Maybe you want to resolve these?

While were at it, I would say lets also disable `Style/IfUnlessModifier` I think this makes the code worse for the human mind.

**Comment by DevinCodes** [comment]
Hey @westonganger, hope you're doing good!

I created a PR to your branch (https://github.com/westonganger/frontman/pull/2) to fix the linting errors. Let me know what you think, so we can move this forward and release this!

Thank you again for all your work on Frontman, we really appreciate it 💯

**Comment by DevinCodes** [comment]
I'll create a PR into your branch to fix the CI today @westonganger 👍

**Comment by DevinCodes** [comment]
Closed in favor of #42

**Pull request closed without merging**